bundles/org.eclipse.osgi/felix/src/org/apache/felix/resolver/Candidates.java

```diff
@@ -18,13 +18,15 @@
  */
 package org.apache.felix.resolver;
 
+import java.io.PrintStream;
 import java.util.*;
 import java.util.Map.Entry;
 import java.util.concurrent.atomic.AtomicBoolean;
 
 import org.apache.felix.resolver.ResolverImpl.ResolveSession;
 import org.apache.felix.resolver.reason.ReasonException;
 import org.apache.felix.resolver.util.*;
+import org.eclipse.osgi.container.ModuleContainer;
```

Comment:
Why is some Equinox specific class used from the felix resolver?

Reply:
I now copied the helper class to the Felix Util